docs/release-notes/whats-new-2.rst

```diff
@@ -445,7 +445,7 @@ and can be used to define DTOs:
 
 - :class:`litestar.dto.dataclass_dto.DataclassDTO`
 - :class:`litestar.dto.msgspec_dto.MsgspecDTO`
-- :class:`litestar.contrib.sqlalchemy.dto.SQLAlchemyDTO`
+- ``litestar.contrib.sqlalchemy.dto.SQLAlchemyDTO``
```

Comment:
I think this doesn't make a lot of sense, since these are supposed to be links. Maybe a link to the usage page would be appropriate here?